creating_packages/package_repo.rst

```diff
@@ -172,8 +172,8 @@ be valid too:
 
 .. tip::
 
-    While you are in the same computer (the same Conan cache), even when you have exported the recipe and
-    Conan has captured the absolute url and commit, Conan will store the local folder where your source code lives.
-    If you build your package locally, it will use the local repository (in the local folder) instead of the remote URL,
-    even if the local directory contains uncommitted changes.
-    This allows you to speed up the development of your packages when cloning from a local repository.
+    When doing a :command:`conan create` of a recipe using ``scm``, Conan will store a local path to the local source repository. Every time
+    you want to create a new package, the sources will not be downloaded from the remote repository but from the local one.
+
+    This allows to build packages making changes to the source code without the need of committing them and pushing them to the remote
+    repository. This convenient to speed up the development of your packages when cloning from a local repository.
```

Comment:
A warning here about the _not-reproducibility_ of packages generated using this _trick_ if there are changes not committed in thee repo?

Reply:
A warning inside a tip? 🤣 Let's see what I can do

Reply:
A mean, warning the user about it, not a _warning box_ 🙈